src/main/scala/io/iohk/ethereum/network/p2p/MessageDecoders.scala

```diff
@@ -43,6 +43,8 @@ object EthereumMessageDecoder extends MessageDecoder {
       //wire protocol
       case (_, Hello.code) => payload.toHello
 
+      //FIXME: I still have an issue with protocolVersion, we are use PV62 and PV63 and code names for Status and NewBlock
+      // suggest that there is PV64, but there isn't
```

Comment:
Hmm now that you say it makes sense to have a PV64

It's weird that ETC has some duplication between the capability from Hello messages and this status field (we even use the same `Versions.PV63` variable), but in copying them we should probably have that duplication as well

It might also be worth researching the difference in purposes between the 2 in case we are missing something

Reply:
Should we research that as part of 280? Or maybe a separate task?